frontend/src/metabase/collections/components/CollectionHeader/CollectionUploadInfoModal.tsx

```diff
@@ -27,15 +27,12 @@ export const UploadInfoModal = ({
       <ModalContent title=" " onClose={onClose}>
         <InfoModalContainer>
           <NewBadge>{t`New`}</NewBadge>
-          <InfoModalTitle>{t`Uploads CSVs to ${applicationName}`}</InfoModalTitle>
+          <InfoModalTitle>{t`Upload CSVs to ${applicationName}`}</InfoModalTitle>
           {isAdmin ? (
             <>
               <InfoModalBody>
                 <p>
-                  {t`Team members will be able to upload CSV files and work with them just like any other data source`}
-                </p>
-                <p>
-                  {t`You'll be able to pick the default database where the data should be stored when enabling the feature.`}
+                  {t`Allow people to upload CSV files to your database so that they can query the CSV data just like any other table.`}
```

Comment:
Mind explaining the theory behind this copy change? I sort of miss the assurance that I have some control over where they'll go in the second line.

Reply:
I waffled on this. Landed on omitting the reference to schema, as I wanted to get people to the setup with as little reading as possible. The schema here is an implementation detail that the setup clearly identifies. 

I also wanted to make it clear that the button was sending people to the setup (vs "Enable in setup", which could be misread as "This button will turn this feature on in the settings"), so that someone would still click on the button just to see the options (if, say, they were wondering whether they could specify the schema).

No strong feelings here so, here's a revision if you prefer:

```suggestion
                  {t`Allow people to upload CSV files to a schema in your database so that they can query the CSV data just like any other table.`}
```

Reply:
I'm still not sure I'm getting a clear sense of what the broader copy update is trying to act on or improve specifically. The original was meant to hit these notes, all of which I think are important but I'm open to improvement on.
- This gives your teammates the ability to add data via CSVs and work with it using the same tools as any other data in the product. I'm not wedded to using teammates the term, but I like the idea of trying to reinforce the need of  "this is meant to help as a relief valve for you, the admin user seeing this"
- Let the setup user know the data goes into their database, but they get control over where if that sounds messy. This mention helps deal with potentially skeptical admins.

I'm game for the button update cause I can see your point about using the word enable here.